### ISSUE TITLE
Pending transaction is not displayed in UI (neither Dapp nor Extension)

### ISSUE THREAD

**Issue by tomusdrw** [paritytech/parity-ui#12]
Error in console:

```
app.js:8 Uncaught BigNumber Error: new BigNumber() not a number: null
```

Topbar and Badge on extension both display "1", but the UI doesn't present the transaction.

**Comment by tomusdrw** [comment]
Triggered by Transaction from Wallet-Contract to Account

**Comment by goldylucks** [comment]
Happens when value is null

**Comment by goldylucks** [comment]
patching in extension, removing once https://github.com/ethcore/parity/issues/1458 is resolved

**Comment by goldylucks** [comment]
published fixed extension v1.0.4
in about an hour should b updated
